lib/rtcpeerconnection.js

```diff
@@ -366,11 +366,18 @@ RTCPeerConnection.prototype._startIceTransport = function _startIceTransport(ice
 /**
  * Add an {@link RTCIceCandidate} to the {@link RTCPeerConnection}.
  * @param {RTCIceCandidate} candidate - the remote ICE candidate
- * @param {?function} [onSuccess]
+ * @param {function} onSuccess
  * @param {?function} [onFailure]
```

Comment:
I think `onFailure` should be required now that we're splitting up the two method signatures.